README.md

```diff
@@ -147,6 +147,84 @@ endif()
 
 `thirdparty/nlohmann_json` is then a complete copy of this source tree.
 
+### Bazel 
```

Comment:
Thanks for the extensive example! What I had in mind, though, was a paragraph like for the package managers in the "Package Managers" section below. Just enough to let people knowing Bazel understand what they need to do to use this library.

Reply:
Since these instructions pertain more to developers, it would probably be better to just download the single header release, not build the entire thing from source. Like so:

```
        http_file(
            name = "com_github_nlohmann_json_single_header",
            urls = [
                "https://github.com/nlohmann/json/releases/download/v3.6.1/json.hpp",
            ],
            sha256 = "blah_blah_blah",
        )
```